app/views/interviewee/_previous_next_doc.html.erb

```diff
@@ -8,7 +8,10 @@
 
       <%= link_to_next_document @search_context[:next] %>
     </div>
+  </div>
 <% end %>
 <% if current_search_session %>
-    <div class="pull-right search-widgets">
-      <%= link_back_to_catalog class: 'btn' %>
\ No newline at end of file
+  <div class="pull-right search-widgets">
```

Comment:
Error: app/views/interviewee/_previous_next_doc.html.erb:14 :: parse error on value "$" ($end) Could not parse app/views/interviewee/_previous_next_doc.html.erb
Location: Could not parse /Users/aprilrieger/softserv/oral-history/app/views/interviewee/_previous_next_doc.html.erb